components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -571,6 +696,162 @@ public void addDefaultEmailTemplates(String tenantDomain) throws I18nEmailMgtExc
         }
     }
 
+    /**
+     * Add the default notification templates which matches the given notification channel to the respective tenants
+     * registry.
+     *
+     * @param notificationChannel Notification channel (Eg: SMS, EMAIL)
```

Comment:
Put the full stop, end of the comment

Reply:
There are several other places with missing full stops. Please add them accordingly.

Reply:
`. `has been added to the method level comment. We do not add `.` at the end of param descriptions.